src/slack/notify.py

```diff
@@ -0,0 +1,17 @@
+import requests
+import json
+
+
+class Notify:
+
+    endpoint = "https://hooks.slack.com/services/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"
```

Comment:
Usually I try to keep this kind of stuff out of my project. Not that it's likely, but if someone found that endpoint, they could post to it. We can regenerate the URL whenever we want, but it would be better to avoid that situation altogether if possible.